python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -321,6 +321,7 @@ def transpile(self,
                 inputs={"X": splited_vars},
                 outputs={},
                 attrs={
+                    "sync_mode": True,
```

Comment:
This seems temporary.
Can you create an issue and assign to you and me? I feel that I need to think a bit more about it.

Reply:
Actually, `ParallelExecuotr` also has a thread pool to parallelism ops, so maybe it's okay for `ParallelExecuotr `, but for the default executor, it not a good solution, it would reduce performance. But if we want to training on multi-CPU or multi-GPU, we need to use `ParallelExecutor` instead of default `Executor`, am I right?

And how about follow issue #10963 and think more about this?

Reply:
Thought might be good to rename `sync_mode` to `wait_rpc`, it's actually not the option for making the training to aync-SGD?